substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/JfrGCHeapSummaryEvent.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please adjust the copyright headers of the files that you added so that they contain the correct year (it seems that you copy/pasted them from other files).

Reply:
@christianhaeubl 

Thank you for pointing that out. 
I was copying and pasting the headers from other files.

I have made some modifications to several files, in addition to the ones I added. Do I need to update the copyright headers of those files as well?

Specifically, the files in question are JfrEvent, JfrFeature, JftType, and JftFileParser.

Reply:
Regarding the existing files that you modified: you can update the copyright headers if you want to but you don't need to.

Reply:
Thank you for reply.
I only modify the copyright headers of the files that I've added.

Reply:
I have completed it.